src/internal/proposal_content.mligo

```diff
@@ -27,8 +27,10 @@ module Types = struct
 
     type 'a t =
     | Transfer of unit transaction
-    | Execute of ('a transaction)
-    | Execute_lambda of { metadata: bytes option; lambda: (unit -> operation) option }
+    | Execute_lambda of
+        { metadata: bytes option
+        ; lambda: (('a ticket) option -> operation * ('a ticket) list) option
+        ; args: ('a * address * nat) option }
```

Comment:
`list` do make more sense, indeed. I created an issue https://github.com/marigold-dev/multisig/issues/20 for record this. It can be improvement. But I'd like to ship this feature ASAP for UI to integrate.